content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,20 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/apply-time-mutation
+
+Type: Annotation
+
+Example: `config.kubernetes.io/apply-time-mutation: "pods"`
+
+Used on: All objects
+
+This annotation is used on the target object to be modified to configure [Apply-Time Mutation](https://github.com/kubernetes-sigs/cli-utils/blob/master/README.md#apply-time-mutation). The annotation may specify one or more substitutions. 
+Each substitution includes a source object, a source field path, and a target field path, with an optional token.
+The source and target field paths are specified using JSONPath.
+If the token is specified, the token is replaced in the target field value string with the source field value. 
+If the token is not specified, the target field value is replaced with the source field value. This alternatively allows either templated interpretation or type preservation.
```

Comment:
You can leave most of this detail out - people can click on the link to learn more.